testUtilIntegration.ts

```diff
@@ -0,0 +1,42 @@
+import { execSync } from "child_process";
+import chalk from "chalk";
+
+// logging
+const { DEBUG } = process.env;
+
+const PREFIX = "mailing tests";
+
+export function error(message?: any, ...optionalParams: any[]) {
+  console.error(chalk.red(PREFIX), message, ...optionalParams);
+}
+
+export function debug(message?: any, ...optionalParams: any[]) {
+  if (DEBUG)
+    console.info(chalk.yellowBright(PREFIX), message, ...optionalParams);
+}
+
+// database cleaning
+const WEB_TABLE_NAMES = ["NewsletterSubscriber"];
+
+const CLI_TABLE_NAMES = ["ApiKey", "Organization", "User"];
+
+export function truncateCliDatabase() {
+  const truncateSql = CLI_TABLE_NAMES.map(
+    (tableName) => `TRUNCATE TABLE "${tableName}" CASCADE;`
+  ).join(" ");
+  debug("Running TRUNCATE on ", process.env.DATABASE_URL_TEST);
+  execSync(`echo '${truncateSql}' | psql ${process.env.DATABASE_URL_TEST}`, {
```

Comment:
@psugihara @elliothursh the tests are flakey with this strategy because TRUNCATE does some asynchronous stuff inside postgres.  Adding a sleep here resolves it, but before I go there do you know if there is a way to check if the command has been committed (like checking an oplog or something) or a way to make it more like a synchronous command like wrapping it in TRANSACTION or something like that?

Reply:
Hm, not sure. An alternative could be running all the truncates then polling each table for 0 count in a loop.